Jenkinsfile

```diff
@@ -135,7 +136,7 @@ try {
 		containers: [
 			containerTemplate(
 				name: 'bfd-cbc-build',
-				image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk11-mvn3-an29-tfenv',
+				image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk11-mvn3-an29-tfenv-aeaa61fa6', //TODO: consider a smarter solution for resolving this image
```

Comment:
Couldn't you tag whichever image we should be currently using `latest`?

Reply:
tl;dr; yes and no. We need to give this just a bit more thought.

We could! However, tag mutability is a double-edged sword. It cuts down on complexity when referencing a `latest` tag, but it complicates things if when changes are introduced that aren't appropriate for all workloads that refer to it. I think the use of tag mutability is still considered somewhat of an anti-pattern.

A reference to a literal, specific image tag isn't _much_ better, because we're left with headspace/complexities surrounding updates to that reference when a new tag is available. A seemingly obvious solution to this would involve storing the desired image tag somewhere, e.g. AWS SSM Parameter. This requires a little more thought, because the awscli doesn't exist in the entrypoint to our Jenkins pipelines... instead, we're looking at potentially storing this in a truly global variable in our Jenkins project.